src/domains/wallet/pages/WalletDetails/components/WalletHeader/WalletHeader.tsx

```diff
@@ -272,15 +269,13 @@ export const WalletHeader = ({
 							</div>
 						</div>
 
-						<div className="flex items-center w-full space-x-5">
-							<span
-								ref={ref}
-								className="flex-1 overflow-hidden text-lg font-semibold text-white w-36 whitespace-nowrap"
-							>
-								<TruncatedAddress />
-							</span>
+						<div className="flex items-center w-full">
+							<TruncateMiddleDynamic
+								value={wallet.address()}
+								className="flex flex-1 text-lg font-semibold text-white whitespace-nowrap"
```

Comment:
```suggestion
								className="w-36 flex-1 text-lg font-semibold text-white whitespace-nowrap"
```
this fix accessible UI and middle truncation component issue (width)
i think you can use `flex-grow` too, i didn't try it